soccer/Processor.cpp

```diff
@@ -402,6 +401,7 @@ void Processor::run() {
         for (NewRefereePacket* packet : refereePackets) {
             SSL_Referee* log = _state.logFrame->add_raw_refbox();
             log->CopyFrom(packet->wrapper);
+            curStatus.lastRefereeTime = packet->receivedTime;
```

Comment:
It appears that the packets are pushed into a vector in the order that they are received but I have no clue if that actually guarantees that the list is always properly sorted.